### DISCUSSION TITLE
exit status

### DISCUSSION THREAD

**Discussion by nandini965** [learndevopsonline/learndevopsonline#838] [Q&A] [answered]
may i know where did i do the mistake, i checked the entire script but couldn't find any mistake ,but not able to get the output
![Screenshot (40)](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/df03c1e4-0532-4f34-b4c1-3e6c220136fd)
![Screenshot (41)](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/fa7ac740-b328-40f2-94c2-28b5e9a0d2fd)
![Screenshot (42)](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/31e62b92-991b-4212-89f6-ce798bcd4f33)
![Screenshot (43)](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/d6c92137-1489-4c20-b667-7df7f50eef8b)

**Comment by anandaiahk** [comment, marked as answer]
to declare function in small s letter but you calling in the function capital S.  func_stat_check (){}          fun_stat_chheck both are using same small leters

**Reply by nandini965** [reply]
yes i got it, thanq
